oz/core/actions.py

```diff
@@ -65,6 +65,13 @@ def init(project_name):
 def server():
     """Runs the server"""
 
+    if oz.settings["logging"]:
```

Comment:
The logging options are automatically available due to [this function](http://www.tornadoweb.org/en/stable/log.html#tornado.log.define_logging_options). It wasn't working when I tried just setting the logging option in our config.py, but after searching through the codebase it looks like I can just call this http://www.tornadoweb.org/en/stable/log.html#tornado.log.enable_pretty_logging